Mathlib/Geometry/Manifold/Algebra/LieGroup.lean

```diff
@@ -21,10 +21,15 @@ groups here are not necessarily finite dimensional.
 ## Main definitions and statements
 
 * `LieAddGroup I G` : a Lie additive group where `G` is a manifold on the model with corners `I`.
-* `LieGroup I G`     : a Lie multiplicative group where `G` is a manifold on the model with
-                        corners `I`.
+* `LieGroup I G` : a Lie multiplicative group where `G` is a manifold on the model with corners `I`.
 * `normedSpaceLieAddGroup` : a normed vector space over a nontrivially normed field
-                                 is an additive Lie group.
+  is an additive Lie group.
+* point-wise inversion and division of maps `M → G` is smooth
```

Comment:
You should specify which declaration(s) provide this fact

Reply:
Well, there are about half a dozen versions of each ({Smooth,ContMDiff}{,On,At}{div,.inv}), which is why I didn't do this at first. I've just referenced the first two.